src/core/Wizard/Step.tsx

```diff
@@ -77,7 +77,8 @@ export const Step = (props: StepProps) => {
         ...style,
       }}
       onClick={onCompletedClick}
-      aria-current={isActive}
+      aria-current={isActive ? 'step' : undefined}
+      tabIndex={0}
```

Comment:
I do not think all he steps should be focusable.. Only the interactive ones, IMO (previous).
Now styling is awful for current and next ones:
![image](https://user-images.githubusercontent.com/81580355/130740252-bc7291fe-7571-457f-869c-5fab2bafa1a1.png)
![image](https://user-images.githubusercontent.com/81580355/130740278-85ba0963-4b93-4e6e-a7e1-7dffce331255.png)
Workflow should not have it at all.

![image](https://user-images.githubusercontent.com/81580355/130740331-63a85ef9-201e-45e8-a9d9-7a15fef5b0ca.png)

Reply:
Also, keyboard click does not work :(

Reply:
Test needed

Reply:
Fixed all